src/overlays/actors/ovl_En_Fish2/z_en_fish2.c

```diff
@@ -823,7 +823,7 @@ void func_80B2A094(EnFish2* this, PlayState* play) {
         Math_Vec3f_Copy(&this->unk_2E8, &sp2C);
     }
 
-    Play_CameraSetAtEye(play, this->unk_2CA, &this->unk_2E8, &this->unk_2DC);
+    Play_SetCameraAtEye(play, this->unk_2CA, &this->unk_2E8, &this->unk_2DC);
```

Comment:
Can name these too.

Reply:
I'll be doing a separate follow-up PR on all the subcam names, which will be naming these variables